src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -73,7 +73,7 @@
         - name: --parent-resource-path
           type: string
           text: The path to the parent resource of the resource being locked.
-        - name: --resource-name
+        - name: --resource
```

Comment:
Can you type `-h` on this command to see if document still works?

Reply:
None of the docs for the parameters here were being used, so I removed them in a new commit